sympy/solvers/solvers.py

```diff
@@ -361,7 +361,51 @@ def checksol(f, symbol, sol=None, **flags):
     # TODO: improve solution testing
 
 
-def check_assumptions(expr, against=None, failing_assumption=False, **assumptions):
+def failing_assumptions(expr, against=None, **assumptions):
+    """
+    Helper for ``check_assumptions``.
+
+    Returns empty list `[]` if all assumptions are True, or a list
+    of  assumption(s) that the expression does not satisfy or
+    `None` if it can't conclude.
+
+    Examples
+    ========
+
+        >>> from sympy import Symbol
```

Comment:
you can dedent the doctest

Reply:
cool. I found indentation in some places of the file, that's why I added here. Are they done for any purpose or should they be removed from everywhere?